### DISCUSSION TITLE
error creating Tracee: failed to load BPF object

### DISCUSSION THREAD

**Discussion by ceeeeb-hao** [aquasecurity/tracee#4107] [Questions and Help]
docker run --name tracee -it --rm   --pid=host --cgroupns=host --privileged   -v /etc/os-release:/etc/os-release-host:ro   -v /var/run:/var/run:ro   -p 3366:3366   aquasec/tracee:latest   --metrics
mkfifo: cannot create fifo '/tmp/tracee/pipe': No such file or directory
starting tracee-ebpf...
libbpf: failed to parse target BTF: -22
libbpf: failed to perform CO-RE relocations: -22
libbpf: failed to load object 'embedded-core'
2024/06/08 12:54:12 error creating Tracee: failed to load BPF object

**Comment by yanivagman** [comment]
This error seems to be from an old version of tracee.
Please pull latest image and try again

**Reply by geyslan** [reply]
It makes sense.

**Reply by ceeeeb-hao** [reply]
Thank you for your answer, this error in docker command may be caused by some recent problems with docker usage within China, so I deployed tracee locally！

**Comment by geyslan** [comment]
@ceeeeb-hao https://github.com/aquasecurity/libbpfgo/discussions/442#discussioncomment-9726283
